plugins/systems/collision/Collision.cpp

```diff
@@ -58,44 +59,38 @@ Systems::ISystem *entryConfig(libconfig::Setting &config)
     return new Systems::Collision(config);
 }
 
-void Systems::Collision::checkPlayerProjectileToEnemyCollision(ECS::Registry &reg) {
-    auto &typeComponents = reg.componentManager().getComponents<Components::Type>();
-    auto &posComponents = reg.componentManager().getComponents<Components::Position>();
-    auto &colliderComponents = reg.componentManager().getComponents<Components::Collider>();
-    auto &damageComponents = reg.componentManager().getComponents<Components::Damage>();
-    auto &healthComponents = reg.componentManager().getComponents<Components::Health>();
+void Systems::Collision::checkPlayerProjectileToEnemyCollision(ECS::Registry &reg)
```

Comment:
Shouldnt the change of this file be done with the checkEnemyProjectileToPlayerCollision ?

Reply:
Of course it should, I just didn't even see that there was another function below this one